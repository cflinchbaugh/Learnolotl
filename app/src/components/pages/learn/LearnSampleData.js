import React from 'react';
import styled from 'styled-components';

import Button from 'buttons/Button';
import Learn from './Learn';

const StyleWrapper = styled.div`

`

function LearnSampleData (props) {
    return (
        <StyleWrapper className="learn-container-wrapper mdCard">
            <Button {...props.cancelSampleButtonData} />
            <Learn {...props.learnData}/>
        </StyleWrapper>
    );
}


export default LearnSampleData;