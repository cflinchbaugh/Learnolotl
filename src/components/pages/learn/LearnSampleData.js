import React from 'react';
import styled from 'styled-components';

import CancelPopButton from 'buttons/CancelPopButton';
import Learn from './Learn';

const StyleWrapper = styled.div`
    .cancel-pop-button-wrapper {
        display: inline-block;
    }
`

function LearnSampleData (props) {
    return (
        <StyleWrapper className="learn-container-wrapper mdCard">
            <div className="cancel-pop-button-wrapper">
                <CancelPopButton {...props.cancelSampleButtonData} />
            </div>

            <Learn {...props.learnData}/>
        </StyleWrapper>
    );
}


export default LearnSampleData;