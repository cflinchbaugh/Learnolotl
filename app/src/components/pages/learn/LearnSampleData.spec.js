import React from 'react';
import {shallow} from 'enzyme';
import LearnSampleData from './LearnSampleData';
import ShallowRenderer from 'react-test-renderer/shallow';

describe('LearnSampleData', () => {
    it('renders and matches snapshot', () => {
        const renderer = new ShallowRenderer();

        const learnSampleDataData = {
            
        }

        const tree = renderer.render(
            <LearnSampleData {...learnSampleDataData}/>
        );

        expect(tree).toMatchSnapshot();
    });

    
});