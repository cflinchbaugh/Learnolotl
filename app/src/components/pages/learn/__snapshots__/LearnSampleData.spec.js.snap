// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LearnSampleData renders and matches snapshot 1`] = `
<styled.div
  className="learn-container-wrapper mdCard"
>
  <Button />
  <Learn />
</styled.div>
`;
